docs/evm_compatibility/json_rpc_api_methods.md

```diff
@@ -1,15 +1,17 @@
 ---
 title: JSON-RPC API Methods
 proofedDate: na
-iterationBy: na
+iterationBy: HB
 includedInSite: true
 approvedBy: na
-comment: 
+comments: TODOs inline 
 ---
 
-*The following tables present the full list of JSON-RPC methods. Each of them is in one of the states:*
+This page presents the full list of JSON-RPC methods. Each of them is in one of three states:
 
-|![done](img/done.ico)| — done|![in progress](img/inprogress.ico) | — in progress|![todo](img/todo.ico) | — to do|
+<!-- todo Andrey can help by providing the deviations fr0m ETH standard methods / Neon equivalent will send 1n Slack todo Neon methods missing -- needs update == Andrey?? -->
```

Comment:
@afalaleev I am returning to the EVM compatibility section PTAL in the discussion you took on providing deviations from Eth standard methods. Is that still with you, or is there someone you want me to ping?

Reply:
@afalaleev pls also confirm you are happy with the "to-do" items -- I think some of these should have a different status.